### DISCUSSION TITLE
we need to implement this asap

### DISCUSSION THREAD

**Discussion by woke-kromcrush-horde** [FarbstoffRSL/RSL-Helper#422] [Ideas]
Its pretty obvious that the company has taken the easy way out with regards to this app - every few huors AND MORE SO during cvc times, the game suspends itself and waits for input to restart.

The excuses they are giving is bs - we all know its to hang up this app and cause people to miss out if they arent buying more of their damn auto play rmt bs.

Im not the most elegant speaker obviously, but everyone here should know what im, sayin g and KNOW the importance of getting around this crappy event.

It shouldnt be that hard to identify a hang up from this pop up and then restart the game and app to continue playing and if we dont implement this then the app here loses 75% of its functionality and I know I will just walk away from rsl all together as its alreayd mroe of a p2w shitshow than i like to be involved with and this app is the only thing keeping it playable for me.

**Comment by Dwza** [comment]
since plarium is aware of the tool and every function is discussed and confirmend by plarium... i dont think the will raise errors to prevent helper from working. its better for them the helper is working because people use way more resources and this brings money to the table. forcing the helper not to work would obviously be a bad idea from plarium because A) they have to pay developers to work against the helper (which functions were confirmed) and B) they would reduce the resource usage.... so i cant fully agree with this statement :D
